catalog/rethinkdb/rethinkdb.bom

```diff
@@ -105,6 +114,17 @@ brooklyn.catalog:
                   - "http://%s:%d"
                   - $brooklyn:attributeWhenReady("host.address")
                   - $brooklyn:config("rethinkdb.http.port")
+          - type: org.apache.brooklyn.enricher.stock.Transformer
+            brooklyn.config:
+              enricher.triggerSensors:
+                - host.address
+                - rethinkdb.cluster.port
+              enricher.targetSensor: $brooklyn:sensor("rethinkdb.cluster.host.and.port")
+              enricher.targetValue:
+                $brooklyn:formatString:
+                  - "%s:%d"
+                  - $brooklyn:attributeWhenReady("host.subnet.address")
+                  - $brooklyn:config("rethinkdb.cluster.port")
```

Comment:
So this seems to no longer be inherited (AMP 4.4) which means the bp never comes up